Source/Classes/SLKTypingIndicatorView.m

```diff
@@ -69,29 +69,50 @@ - (void)slk_commonInit
     self.highlightFont = [UIFont boldSystemFontOfSize:12.0];
     self.contentInset = UIEdgeInsetsMake(10.0, 40.0, 10.0, 10.0);
     
-    self.backgroundColor = [UIColor whiteColor];
-    
     [self addSubview:self.textLabel];
     
     [self slk_setupConstraints];
 }
 
 
-#pragma mark - UIView Overrides
+#pragma mark - SLKTypingIndicatorProtocol
+
+- (void)setVisible:(BOOL)visible
+{
+    // Skip when updating the same value, specially to avoid inovking KVO unnecessary
+    if (visible == self.isVisible) {
+        return;
+    }
+    
+    // Required implementation for key-value observer compliance
+    [self willChangeValueForKey:NSStringFromSelector(@selector(isVisible))];
```

Comment:
Synthesizing (explicitly or implicitly) this property will automatically make it KVO compliant.
However, the `key` will be "visible" and not "isVisible"